unlock-app/src/services/storageService.ts

```diff
@@ -71,13 +70,27 @@ export class StorageService extends EventEmitter {
     return this.locksmith.login(message, signature)
   }
 
+  async signOut() {
+    const endpoint = `${this.host}/v2/auth/revoke`
+    const response = await fetch(endpoint, {
```

Comment:
Let's make this fail silently (try/catch) since it should not affect the rest of the app if it does.